engine/gamesys/src/waf_gamesys.py

```diff
@@ -303,10 +303,15 @@ def transform_gui(task, msg):
     msg.script = msg.script.replace('.gui_script', '.gui_scriptc')
     font_names = set()
     texture_names = set()
+    material_names = set()
+
     msg.material = msg.material.replace(".material", ".materialc")
     for f in msg.fonts:
         font_names.add(f.name)
         f.font = f.font.replace('.font', '.fontc')
+    for f in msg.materials:
+        material_names.add(f.name)
+        f.material = f.material.replace('.material', '.materialc')
```

Comment:
Could we remove waf_gamesys.py and use bob for building test content? Feels really strange that we have yet another way of building content.. :(